libcontainer/process_linux.go

```diff
@@ -357,6 +359,10 @@ func (p *initProcess) start() error {
 			}
 			sentRun = true
 		case procHooks:
+			// Setup cgroup before prestart hook, so that the prestart hook could apply cgroup permissions.
+			if err := p.manager.Set(p.config.Config); err != nil {
+				return newSystemErrorWithCause(err, "setting cgroup config for ready process")
+			}
```

Comment:
NIT.  Can we have slightly different error message text so it can be more easily determined where the error came from?  Perhaps appending " in procReady" to the first message at line 339 and " in procHooks" here in the second?